install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -220,7 +220,9 @@ spec:
                 done
               else
                 # COS-beta (with containerd)
-                for f in `find /var/lib/cni/networks/ -type f ! -name lock ! -name last_reserved_ip.0`; do crictl stopp $(cat $f) || true; done
+                PATH=${PATH}:{{ .Values.cni.binPath }}
+                which crictl
```

Comment:
How does it help running "which crictl"? Is it to make the script existing since it couldn't find such command and exit the node-init?

Reply:
I just found it useful debugging information. I can remove if you don't believe it is relevant to keep.

Reply:
Removed it as I am now calling crictl using the absolute path.